### PR TITLE
Solved the import warnings and its already TS

### PR DESCRIPTION
**Pull request by GithinjiHans**

### CODE REVIEW

File: src/declaration.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+declare module '*.png';
```
Comment (Tushar-4781) [inline src/declaration.d.ts:1]: what about the images having extension svg ?

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Comment by pulkitvyas08** [comment]
@GithinjiHans "Convert MyFeelings into Typescript" @Tushar-4781 meant that the appropriate types and interfaces are not defined on the MyFeelings page. I am supposed to fix that since I've been working on it. So once you resolve the issues raised by @Tushar-4781, we can merge your PR but won't close #285

File: src/declaration.d.ts (hunk 1 above)
Comment (pulkitvyas08) [inline src/declaration.d.ts:1]: Maybe change the file name to `global.d.ts` instead. Also, add a blank line at the end of the file

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]

File: src/declaration.d.ts (hunk 1 above)
Comment (GithinjiHans) [reply src/declaration.d.ts]: I've added the modules

Comment (GithinjiHans) [reply src/declaration.d.ts]: ok

**Comment by GithinjiHans** [comment]
I've made another pull request....Merge that then I'll be able to make this deploy

File: src/global.d.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+declare module '*.png';
+declare module '*.svg';
+declare module '*interfaces';
+declare module '*ILanguage';
+declare module '*FeelingsAPI';
```
Comment (pulkitvyas08) [inline src/global.d.ts:5]: Add a blank line at the end of this file

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]

**Comment by pulkitvyas08** [comment]
Vercel deployment is also not working. Cannot give a go-ahead before that is fixed

**Comment by GithinjiHans** [comment]
@Tushar-4781 I've solved the deployment error would you mind reviewing it ...

**Review by pulkitvyas08 — APPROVED** [review]

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**